wagtail/users/views/users.py

```diff
@@ -70,74 +70,70 @@ def get_users_filter_query(q, model_fields):
     return conditions
 
 
-@any_permission_required(add_user_perm, change_user_perm, delete_user_perm)
-@vary_on_headers("X-Requested-With")
-def index(request, *args):
-    q = None
-    is_searching = False
-
-    group = None
-    group_filter = Q()
-    if args:
-        group = get_object_or_404(Group, id=args[0])
-        group_filter = Q(groups=group) if args else Q()
+class Index(IndexView):
+
```

Comment:
Not a huge problem but it is nice to add some kind of Pydoc as we go - while these do not end up in any documentation (yet), it does make it easier to scan through this file.


```suggestion
    """
    Lists the users for management within the admin.
    """

```